### ISSUE TITLE
User having difficulty navigating between pages on screenreader

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/platform-forms-client#6015]
## Description

https://gcdigital.slack.com/archives/C03T4FAE9FV/p1755197812363839

A client trying to transform a multi-page PDF form into a new GC Form reached out with a question.

Firstly, using a screenreader, they click the “New Page” button at the bottom of the "Start page" in GC Forms, and are brought to a new page, with fields for “Page Name” and “Page title”, which
is confusing for a screenreader user without context. They are unsure what to input into these two fields.

Secondly, as a screen reader user, they are unsure how to navigate between pages successfully, as they do not notice a clear method in which to view all the pages in the form and how
to navigate back to the first page and vice-versa.

> Confusing areas: 
> - How would one navigate between pages, especially if one is a screen reader user? 
> - What are the differences between the “Page Name” and “Page Title” fields? and what kind of information must be entered here?
> - On a side note, is there training one can take to better understand GC Forms, especially if one is a screen reader user?

## Potential solutions

- Test on Jaws/NVDA on windows
- Re-look and make sure the page setup hidden link is fully working...seems it does not link when the nav is closed currently, also takes a lot of tabbing to reach it
- Have some sort of "Add element" show up after you add a page to make that more obvious
- Investigate possibility of having the panel stick open in Edit mode (open by default) and only close if it's shut off intentionally (rather than closing everytime you navigate elsewhere)
- Add "hot keys in the app to make jumping between the panel and the editor faster and easier to go back and forth
- Describe "Form name" and "Page name" as internal references or unilingual name you see as you work on the form in the sidebar or in the branching view
- Describe "Form title" and "Page title" as what will be visible to end users and will need to be bilingual by providing EN-FR equivalents in the Translations section
- Make the "Forms set-up" panel more obvious, especially for assistive technology users, where pages are listed in the sidebar and where/how you can navigate between them

**Comment by anikbrazeau** [comment]
From client / screenreader user: https://cds-snc.freshdesk.com/a/tickets/23974

> After a bit more exploration, I have determined that the tree view for the pages is not at all useable or accessible when using JAWS. If one uses NVDA, although it does appear NVDA reads the pages, it’s extremely difficult to delete pages. 

> This may be something the GC Forms developers may want to bring back to the drawing board and make this area more accessible with all screen readers.
